frontend/src/components/Room/CreateRoom.js

```diff
@@ -107,13 +112,24 @@ const CreateRoom = () => {
                             rules={[
                                 {
                                     required: true,
-                                    message: "Please enter room name.",
+                                    max: 20,
                                 },
                             ]}
                         >
                             <Input placeholder="Enter room name" />
                         </Form.Item>
 
+                        <Form.Item
+                            name="description"
+                            label="Description :"
+                            id="description"
```

Comment:
```suggestion
                            id="description"
```

Is this **id** needed?

Reply:
ma htae ll ya tal. ma use voo

Reply:
> Is this **id** needed?

ma htae ll ya tal. ma use voo